<?php
    
try {
    // Something goes wrong
} catch (MySpecialException) {
    Log::error("Something went wrong");
}